ArmPkg/Library/TpmServiceStateTranslationLib/TpmServiceStateTranslationLib.c

```diff
@@ -0,0 +1,646 @@
+/** @file
+  Implementation for the TPM Service State Translation Library
+
+  Copyright (c), Microsoft Corporation.

```

Comment:
If there is code inherited from open source version, please include the original copyright.

Reply:
The same for other files involved in this PR.

Reply:
The same for other files involved in this PR.